### ISSUE TITLE
MSSql: support windows authentication

### ISSUE THREAD

**Issue by n-riesco** [plotly/falcon-sql-client#410]
Right now, we're using [tedious](https://www.npmjs.com/package/tedious) a pure Javascript TDS driver. Although there is work in progress to support windows authentication (on windows only). See:
- https://github.com/tediousjs/tedious/issues/415
- https://github.com/tediousjs/tedious/issues/246
- https://github.com/tediousjs/tedious/pull/497
- https://github.com/tediousjs/tedious/commit/f5a2260f
- https://github.com/tediousjs/tedious/pull/624
- https://github.com/tediousjs/tedious/issues/512
- https://github.com/tediousjs/tedious/issues/690

---

Alternative drivers:
- [msnodesqlv8](https://github.com/TimelordUK/node-sqlserver-v8) (windows only, native module, requires Visual Studio 2015/2017)

**Comment by n-riesco** [comment]
Re #300

**Comment by pabucur** [comment]
@n-riesco  Any progess on the Windows Authentification issue?

**Comment by n-riesco** [comment]
@pabucur No, no progress. I've just checked and I haven't seen any work on mssql in https://github.com/tediousjs/tedious since April.

**Comment by charleyferrari** [comment]
Another existing Plotly client has also requested this feature.

cc @n-riesco @nicolaskruchten @sudburyrob
